app/src/main/res/values/dimens.xml

```diff
@@ -3,17 +3,28 @@
     <dimen name="dp_50">50dp</dimen>
     <dimen name="dp_8">8dp</dimen>
     <dimen name="dp_5">5dp</dimen>
+    <dimen name="dp_2">2dp</dimen>
     <dimen name="dp_10">10dp</dimen>
     <dimen name="dp_16">16dp</dimen>
     <dimen name="dp_24">24dp</dimen>
     <dimen name="dp_22">22dp</dimen>
+    <dimen name="dp_36">36dp</dimen>
     <dimen name="dp_40">40dp</dimen>
+    <dimen name="dp_45">45dp</dimen>
     <dimen name="dp_52">52dp</dimen>
     <dimen name="dp_85">85dp</dimen>
+    <dimen name="dp_70">70dp</dimen>

```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_70` appears to be unused
 Unused resources make applications larger and slow down builds.